client/src/Upload.js

```diff
@@ -51,36 +54,55 @@ const styles = makeStyles(theme => ({
         marginRight: theme.spacing(1),
         width: 500,
     },
-}));
+});
 
 const handleSubmit = () => {
     const file = document.getElementById('imagePicker')
     uploadFile(file.files[0])
 }
 
-export default function Upload() {
+class Upload extends React.Component {
+    constructor(props) {
+        super(props);
+        
+        this.state = {
+
+            uid: "",
+            recipeName: "",
+            mealTypeArray: ["breakfast", "lunch", "dinner"],
```

Comment:
There are 6 meal types. See ZenHub for more details: https://app.zenhub.com/workspaces/delight-5d85617c3b0ef90001a402e4/issues/cs157a-team18/cs157a-team18/55.

Also, if you are unable to change this array in the format specified in the ZenHub, maybe you can make `mealTypeArray` store mealType objects?

```
mealTypeArray: [{id: 1, type: "breakfast"}, {id: 4, type: "lunch"}, {id: 3, type: "dinner"}...]
```

Reply:
fixed